Sources/Vapor/HTTP/Server/HTTPServer.swift

```diff
@@ -357,7 +380,10 @@ private final class HTTPServerConnection: Sendable {
             }
             
             // Set the handlers that are applied to the accepted Channels
-            .childChannelInitializer { [unowned application] channel in
+            .childChannelInitializer { [unowned application, unowned server] channel in
+                /// Copy the most up-to-date configuration.
```

Comment:
Nit - this doesn't need to be a doc comment
```suggestion
                // Copy the most up-to-date configuration.
```

Reply:
I disagree with Tim here - I also use doc comments most of the time (though more for the inline formatting than for a font per se xD), I'd be more inclined to change the existing comments rather than the new one 🙂

Reply:
Happy to revert them, let me know 🤣

Reply:
Updated! Let me know if you'd rather them in a separate PR for commit history reasons, otherwise good to merge if you are happy with it.